### ISSUE TITLE
nudge operation failed, verify permissions and parameters.

### ISSUE THREAD

**Issue by jplopezy** [googleprojectzero/winafl#180]
Good!

Excuse me, but here I am with new problems.

I was trying a harness that modified a bit (it was a code that I found out there to play music)

The code is not very complex, in fact just add charToWChar method and some other things and that's it.

```

include "sound.h"


wchar_t* charToWChar(const char* text)
{
size_t size = strlen(text) + 1;
wchar_t* wa = new wchar_t[size];
mbstowcs(wa, text, size);
return wa;
}



int main(int argc, char** argv)
{
sound player;

player.OpenFromFile(charToWChar(argv[1]));

player.Play();

while (player.GetPosition() < 1 ) {
printf("Playing.. : %f elapsed\n",player.GetPosition());

}


player.Close();


return 0;
}

```

The problem is that winafl returns the following error:

[-] PROGRAM ABORT : Test case 'id_000000' results in a timeout
         Location : perform_dry_run(), c:\work\winafl\source\afl-fuzz.c:2951

0 processes nudged
nudge operation failed, verify permissions and parameters.

and drrun

Module loaded, MSACM32.DRV
Module loaded, MIDIMAP.dll
WARNING: Post-fuzz handler was never reached. Did the target function return normally?
Coverage map follows:

In summary I have my doubts if something in the harness is wrong or in the dyna.

I read that there is a currently known issue (this must be the reason why I can not give it more the name of the function and only the offset)

I would like to know how to solve the topic of target function, even if he passes the corrected offset (with the outbound search for the main function) but generates that error ...

Anyone think of something?

regards

**Comment by jplopezy** [comment]
sorry (false positive)

**Comment by xinyile** [comment]
Hi, do you solve the problem that Post-fuzz handler was never reached? I also meet this problem now and am not quite sure what I should do with target function.
